src/sentry/sentry_metrics/indexer/cloudspanner_model.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+from dataclasses import dataclass
+from typing import Sequence
+
+_TIMESTAMP_TZ_FORMAT = "%Y-%m-%d %H:%M:%S %Z"
+
+
+@dataclass
+class SpannerIndexerModel:
+    id: int
+    string: str
+    organization_id: int
+    date_added: datetime.datetime
+    last_seen: datetime.datetime
+    retention_days: int
+
+    def to_values_format_dml(self) -> str:
+        """
+        Returns a string in the format of (id, string, organization_id,
+        date_added, last_seen, retention_days) which can be used in the
+        VALUES section of INSERT statement.
+        """
+
+        return (
+            f'({self.id}, "{self.string}", {self.organization_id},'
```

Comment:
1. won't this allow sqli?
2. double-quotes in postgres sql mean identifiers. i don't have enough context on dml to understand, but in the next line you're using single-quotes for datetime string

Reply:
Good point. Let me make this safer.

Reply:
I have removed the use of double-quotes and use single-quotes everywhere now. I have added a TODO for sql injection. I wasn't able to find a good way to enforce this in the short time period. We can come back to it later it that's ok.

Reply:
i feel iffy about this. if this followup work slips through the cracks we have a serious security incident on our hands.

Reply:
Using parametrized SQL now.